scripts/engineering/aggregate_phase44d_carry_aware_experimental_3x3_scaling.py

```
@@ -0,0 +1,191 @@
+    benchmark_version = None
+    semantic_scope = None
+    timing_unit = None
+    canonical_rows: list[dict[str, Any]] | None = None
+    timing_samples: dict[tuple[str, str, int], dict[str, list[float]]] = {}
+
+    for input_path in args.inputs:
+        payload = json.loads(input_path.read_text(encoding="utf-8"))
+        if payload.get("timing_mode") != EXPECTED_INPUT_TIMING_MODE:
+            raise SystemExit(
+                f"{input_path} must be a {EXPECTED_INPUT_TIMING_MODE!r} benchmark payload; got {payload.get('timing_mode')!r}"
+            )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)